composer.json

```diff
@@ -43,13 +43,24 @@
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "scripts": {
         "check": [
             "@test",
             "@cs-check"
         ],
         "cs-check": "phpcs --colors -p ./src ./tests",
         "cs-fix": "phpcbf --colors -p src/ tests/",
-        "test": "phpunit --colors=always"
+        "test": "phpunit --colors=always",
+        "stan": [
+            "@phpstan",
+            "@psalm"
+        ],
+        "phpstan": "tools/phpstan analyse",
+        "psalm": "tools/psalm --show-info=false",
+        "stan-baseline": "tools/phpstan --generate-baseline",
```

Comment:
Should we just call this phpstan-baseline since `stan` it a shortcut for all analyzers?